src/command/backup/backup.c

```diff
@@ -720,11 +720,11 @@ backupResumeFind(const Manifest *manifest, const String *cipherPassBackup)
                                     strZ(manifestData(manifest)->backrestVersion), strZ(manifestResumeData->backrestVersion));
                             }
                             // Check backup type because new backup label must be the same type as resume backup label
-                            else if (manifestResumeData->backupType != backupType(cfgOptionStr(cfgOptType)))
+                            else if (manifestResumeData->backupType != cfgOptionStrId(cfgOptType))
```

Comment:
No, because this is simple equality. In the same way uint32 == uint64 is OK, because we are not trying to assign from int64 from int32, which might result in truncation.

Reply:
Sorry, meant *from int64 to int32*.